Transports features summary (#4397)
Particular/docs.particular.net: transports/azure-service-bus/index.md

[Azure Service Bus](https://azure.microsoft.com/en-us/services/service-bus/) is a messaging service hosted on the Azure platform that allows for exchanging messages between various applications in a loosely coupled fashion. The service offers guaranteed message delivery and supports a range of standard protocols (e.g. REST, AMQP, WS*) and APIs such as native pub/sub, delayed delivery, and more.

> ## Transport facts

Reviewer: ```suggestion
## Transport at a glance
```

|Feature                    |   |  
|:---                       |---
|Transactions |None, ReceiveOnly, SendsWithAtomicReceive
|Pub/Sub                    |Native
|Timeouts                   |Native

> |Large message bodies       |via higher tier (i.ex. Premium) or DataBus

Reviewer 2: ```suggestion
|Large message bodies       | with Premium tier or DataBus
```
